src/main/resources/assets/styles/_project.scss

```diff
@@ -0,0 +1,82 @@
+.project {
+  .title-max-width {
+    max-width: 680px;
+  }
+
+  .title-padding {
+    padding: 20px 0px;
+  }
+
+  .project-details {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 1.2rem 0;
+    padding-bottom: 1.5rem;
+    border-bottom: 1px solid #c3dcdc;
+
+    .details-block {
+      width: 33%;
+      padding-top: 1.5rem;
+      border-top: 1px solid #c3dcdc;
+      display: flex;
+
+      > div {
+        margin-left: .5rem;
+        font-family: $font-open-sans;
+        font-size: 20px;
+        line-height: 1.6;
+
+        > a {
+          font-weight: 800;
+        }
+
+        > span {
+          font-weight: 700;
+        }
+      }
+    }
+  }
+
+  .project-ingress {
+    > p {
+      font-weight: 400;
+      font-size: 20px;
+      line-height: 32px;
+      max-width: 680px;
+      margin-bottom: 0px;
+      margin-left: auto !important;
+      margin-right: auto !important;
+    }
+  }
+
+  .project-paragraph {
+    > p {
+      font-size: 16px;
+      font-stretch: normal;
+      font-weight: 400;
+      line-height: 28px;
+      max-width: 680px;
+      margin-left: auto !important;
+      margin-right: auto !important;
+    }
+  }
+
+  .hide-empty-block {
+    display: none;
+  }
+
+  @media (min-width: 1024px) {
+    .center {
+      margin-left: auto !important;
+      margin-right: auto !important;
+    }
+  }
+
+  @media (max-width: 1024px) {
+    .top-padding {
+      padding-top: 30px;
+      padding-bottom: 10px;
+    }
+  }
```

Comment:
Vi bør bruke Bootstrap sine innebygde funksjoner for å gjøre slike ting som dette.
https://getbootstrap.com/docs/5.0/layout/grid/

Reply:
https://getbootstrap.com/docs/5.0/layout/gutters/#no-gutters

Reply:
https://getbootstrap.com/docs/5.0/utilities/spacing/